### PR TITLE
libraryapp_userservice

### PR DESCRIPTION
**Pull request by saikumar2912**

### CODE REVIEW

File: libraryapp/service/userservice.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            fields: ["_id", "name", "email"]
+        };
+        const dbUserName = "apikey-v2-380rhhqzbqk6eifbn30gvuevzk9903pdrrsd7f8ipknj";
+        const dbPassword = "ee0e39016c30dc0fc4fd04d12a420174";
+        const basicAuth = 'Basic ' + btoa(dbUserName + ':' + dbPassword);
+        let url = "https://5804af1c-53d6-4cc1-b0eb-5219a1cc5775-bluemix.cloudantnosqldb.appdomain.cloud/libraryapp_users/_find";
+        return (axios.post(url, selectedData, { headers: { 'Authorization': basicAuth } }));
+    }
+
+    static register(formData) {
```
Comment (Renis8903) [inline libraryapp/service/userservice.js:18]: Add comments to methods

File: libraryapp/service/userservice.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class UserService {
+    static login(userName, password) {
```
Comment (fresher-batch-2021) [inline libraryapp/service/userservice.js:3]: Add Comments

Comment (saikumar2912) [reply libraryapp/service/userservice.js:3]: Done

File: libraryapp/service/userservice.js (hunk 1 above)
Comment (saikumar2912) [reply libraryapp/service/userservice.js:18]: Done

**Review by Renis8903 — APPROVED** [review]
Approved

**Pull request merged**